en_li_vinigret.py

```diff
@@ -0,0 +1,23 @@
+#submitted by Keren Or Hadad 208025205
+import datetime
+import random
+
+
+def en_li_vinigret(start_date: datetime, end_date: datetime):
```

Comment:
Please add documantation to your function.

Reply:
done